prog/dftb+/lib_dftbplus/inputdata.F90

```diff
@@ -29,6 +29,7 @@ module dftbp_inputdata
 #:endif
   use dftbp_pmlocalisation, only : TPipekMezeyInp
   use dftbp_elstatpot, only : TElStatPotentialsInp
+  use dftbp_reksvar, only : TReksIni
```

Comment:
@aradi I have one problem for importing reks modules with `dftbp_reks`. There is no problem for inputdata.F90 and initprogram.F90, but circular dependency appears for mainio.F90 file. Since `dftbp_reksen`, `dftbp_reksinterface` and `dftbp_reksproperty` modules shows `use mainio`, thus the conflict between mainio and dftbp_reks occurs when I change `use dftbp_reksvar` in mainio.F90 to `dftbp_reks`.

Reply:
I see. OK, then leave it for the moment as it is, I'll have a look, whether we can fix it somehow.

Reply:
Done with commit c2bdedc except mainio-related modules.